mars/deploy/local/tests/test_cluster.py

```diff
@@ -196,6 +196,15 @@ def testIndexTensorExecute(self):
 
                 np.testing.assert_array_equal(r, np.ones((5, 5)) * 2)
 
+            with new_session(cluster.endpoint) as session3:
+                a = mt.ones((100, 5))
+
+                slice1 = a[:10]
+                slice2 = a[10:20]
+                r1, r2 = session3.run(slice1, slice2)
```

Comment:
This test is meaningless, should test on random data, if test on ones, r1 will always be equal to r2 whatever the generated key is same or not.